rocky/katalogus/client.py

```diff
@@ -205,6 +209,15 @@ def get_cover(self, boefje_id: str) -> BytesIO:
         response.raise_for_status()
         return BytesIO(response.content)
 
+    def create_plugin(self, plugin: Plugin) -> None:
+        response = self.session.post(
+            f"{self.organization_uri}/plugins",
+            headers={"Content-Type": "application/json"},
+            content=plugin.model_dump_json(exclude_none=True),
+        )
+        logger.info("Created new plugin %s", plugin.name)
```

Comment:
The API specifically states a 201 status code to indicate that the resource is created